app/views/patients/show.html.erb

```diff
@@ -47,6 +47,9 @@
           <div class="level-item">
```

Comment:
The Health Card Details page (linked from the top of this file) shows no code or value field in the Observation resource. I decoded the downloaded `.smart-health-card` file and it also does not contain the status and it also only contains the status and date of the observation.

Reply:
resolved